recipe/meta.yaml

```diff
@@ -92,6 +92,7 @@ requirements:
     - {{ compiler('cxx') }}
     - {{ cdt('xorg-x11-proto-devel') }}  # [linux]
     - {{ cdt('libx11-devel') }}          # [linux]
+    - {{ cdt('libbluetooth-dev') }}      # [not win]
```

Comment:
```suggestion
    - {{ cdt('libbluetooth-dev') }}      # [linux]
```

Reply:
This is something I found out later. it's `libbluetooth-dev` for debian / ubuntu, but `bluez-dev` for things like alpine distributions. So I have no idea on how to specify things on a per distro.